src/python_testing/TC_DRLK_2_9.py

```diff
@@ -0,0 +1,566 @@
+        validpincodestr = await self.generate_pincode(self.maxpincodelength)
+        self.pin_code2 = bytes(validpincodestr, 'ascii')
+        inavlidpincodestr = await self.generate_pincode(self.maxpincodelength+1)
+        self.inavlid_pincode = bytes(inavlidpincodestr, 'ascii')
+        validpincodestr = await self.generate_pincode(self.maxrfidcodelength)
+        self.rfid_tag = bytes(validpincodestr, 'ascii')
+
+    async def send_clear_user_cmd(self, user_index, expected_status: Status = Status.Success):
+        try:
+            await self.send_single_cmd(cmd=drlkcluster.Commands.ClearUser(userIndex=user_index),
+                                       endpoint=self.app_cluster_endpoint,
+                                       timedRequestTimeoutMs=1000)
+            asserts.assert_equal(expected_status, Status.Success)
+        except InteractionModelError as e:
+            asserts.assert_equal(e.status, expected_status, f"Unexpected error returned: {e}")
+
+    async def get_user(self, userindex, username, useruniqueid, credentiallist, expected_status: Status = Status.Success):
+        try:
+            response = await self.send_single_cmd(cmd=drlkcluster.Commands.GetUser(userIndex=userindex),
+                                                  endpoint=self.app_cluster_endpoint,
+                                                  timedRequestTimeoutMs=1000)
+            asserts.assert_true(type_matches(response, drlkcluster.Commands.GetUserResponse),
+                                "Unexpected return type for GetUserResponse")
+            asserts.assert_true(response.userIndex == userindex,
+                                "Error when executing GetUserResponse command, credentialExists={}".format(
+                                    str(response.userIndex)))
+            asserts.assert_true(response.userName == username,
+                                "Error when executing GetUserResponse command, credentialExists={}".format(
+                                    str(response.userName)))
+            asserts.assert_true(response.userUniqueID == useruniqueid,
+                                "Error when executing GetUserResponse command, credentialExists={}".format(
+                                    str(response.userUniqueID)))
+            asserts.assert_true(response.credentials == credentiallist,
+                                "Error when executing GetUserResponse command, credentialExists={}".format(
+                                    str(response.credentials)))
+            asserts.assert_equal(expected_status, Status.Success)
+
+        except InteractionModelError as e:
+            asserts.assert_equal(e.status, expected_status, f"Unexpected error returned: {e}")
+
+    async def get_credentials_status(self, credentialIndex: int, credentialType: drlkcluster.Enums.CredentialTypeEnum,
+                                     step, userIndex, credential_exists):
+        if step:
+            self.step(step)
```

Comment:
Remove this redundant check as most of the steps are being printed/called before this function is called